modules/caddyhttp/rewrite/rewrite_test.go

```diff
@@ -323,7 +333,6 @@ func TestRewrite(t *testing.T) {
 			input:  newRequest(t, "GET", "/foo/findme%2Fbar"),
 			expect: newRequest(t, "GET", "/foo/replaced%2Fbar"),
 		},
-
```

Comment:
I usually leave a line break sometimes to weakly organize test cases into groups. It'd be convenient if we could preserve this line break.